pkg/lua/funcs.go

```diff
@@ -0,0 +1,35 @@
+package lua
+
+import (
+	"k8s.io/apimachinery/pkg/api/meta"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+type Status struct {
+	Conditions []metav1.Condition
+}
+
+func isStatusConditionExist(s map[string]interface{}, condition string) bool {
```

Comment:
perhaps rename to `statusConditionExists`

Reply:
done